platform/darwin/app/wideline.metal

```diff
@@ -0,0 +1,495 @@
+                // intersect, as the distance to the original intersection point isn't helpful.
+                // todo: this doesn't make sense for small angles, but what's the actual threshold?
+                if (centerLine != 0) {
+                    const float2 cn0 = cur.norm * centerLine * screenScale;
+                    const float2 cn1 = next.norm * centerLine * screenScale;
+                    const IntersectInfo i2 = intersectLines(prev.screenPos + cn0, cur.screenPos + cn0,
+                                                            cur.screenPos + cn1, next.screenPos + cn1);
+                    if (i2.valid && length_squared((i2.interPt - offsetCenter)/screenScale) < maxAdjDist*maxAdjDist) {
+                        offsetCenter = i2.interPt;
+                    }
+                }
+                
+                const float2 interVec = (interPt - offsetCenter) / screenScale;
+                const float interDist2 = length_squared(interVec);
+                const float maxClipDist2 = (maxAdjDist * wideVec.interClipLimit) *
+                                           (maxAdjDist * wideVec.interClipLimit);
+
+                // Limit intersection distance.
+                // For up to a multiple of that, adjust the intersection point back along its
+                // length to that limit, effectively narrowing the line instead of just giving up.
+                if (interDist2 <= maxAdjDist*maxAdjDist) {
+                    intersectValid = true;
+                } else if (wideVec.interClipLimit > 0 && interDist2 <= maxClipDist2) {
+                    interPt = offsetCenter + normalize(interVec) * sqrt(interDist2) * screenScale;
+                    intersectValid = true;
+                }
+            }
+        }
+    }
+
+    // Endcaps not used for miter case, discard them.
+    if ((joinType == WKSVertexLineJoinMiter || joinType == WKSVertexLineJoinMiterSimple) &&
+        whichPoly != WideVecPolyBodyGeom && !isStartCap && !isEndCap) {
+        return outVert;
+    }
+
+    // Note: We're putting a color in the instance, but then it's hard to change
+    //  So we'll pull the color out of the basic drawable
+    float4 color = vert.color;
+
+    outVert.color = color;
+    outVert.w2 = w2;
+    outVert.edge = wideVec.edge;
+
+    // Work out the corner positions by extending the normals
+    const float2 realEdge = interSgn * w2 * screenScale;
+    const float2 corner = offsetCenter + centers[2].norm * realEdge;
+    const float turnSgn = turningLeft ? -1 : 1;
+    const bool isInsideEdge = (isLeft == turningLeft);
+
+    // Current corner is the default result for all points.
+    float2 pos = corner;
+
+    // Texture position is based on cumulative distance along the line.
+    // Note that `inst[2].totalLen` wraps to zero at the end, and we don't want that.
+    // Texture coords are used for edge blending, so we need to set them up even if there are no textures.
+    float texY = 0;
+    float texX = 0;
```

Comment:
It looks like texture support was removed from this shader, I see some texcoord manipulation commented out in a few places. Could you remove these references?